vscode/src/extension.ts

```diff
@@ -8,7 +8,7 @@ import {
   log,
   getLibrarySourceContent,
   qsharpLibraryUriScheme,
-} from "qsharp";
+} from "qsharp-lang";
```

Comment:
So no package aliasing? We're just committing to the new name?

Reply:
Yep. Being it will be 'qsharp-lang' on npm, I think aliases would just be confusing. We should use it as anyone installing from npm will.